registry/tracecat_registry/base/core/http.py

```diff
@@ -16,6 +16,7 @@
 ssl_secret = RegistrySecret(
     name="ssl",
     optional_keys=["SSL_CLIENT_CERT", "SSL_CLIENT_KEY", "SSL_CLIENT_PASSWORD"],
```

Comment:
```suggestion
    keys=["SSL_CLIENT_CERT"],
    optional_keys=["SSL_CLIENT_KEY", "SSL_CLIENT_PASSWORD"],
```

Has this combo been tested (i.e. `optional=True` with non null keys and optional keys?)

Reply:
added more tests, and see:
https://github.com/TracecatHQ/tracecat/pull/508/files#diff-c466a7ce56c24160eb03320ad4f34f491c4482b72aa6d44335ad3d1bcc85f29eR74-R92

^ at commit time if a secret is optional but it's defined, we check that all required keys are defined, otherwise return validation errors